docker/keeper/Dockerfile

```diff
@@ -44,7 +44,10 @@ ARG DIRECT_DOWNLOAD_URLS=""
 # We do that in advance at the begining of Dockerfile before any packages will be
 # installed to prevent picking those uid / gid by some unrelated software.
 # The same uid / gid (101) is used both for alpine and ubuntu.
-
+ARG DEFAULT_UID="101"
+ARG DEFAULT_GID="101"
+RUN addgroup -S -g "${DEFAULT_GID}" clickhouse && \
+    adduser -S -h "${DEFAULT_DATA_DIR}" -s /bin/bash -G clickhouse -g "ClickHouse keeper" -u "${DEFAULT_UID}" clickhouse
```

Comment:
1. $DEFAULT_DATA_DIR is not defined here yet
2. Home for the `clickhouse` user intentionally set to `/var/lib/clickhouse` to be the same globally. And I can't say quickly, what should we do here.

Reply:
1. Oops. Tried a few formats and I guess I missed that bit in the last change
2. Fair enough, changed back